website/static/js/registrationUtils.js

```diff
@@ -32,59 +32,60 @@ var currentUser = window.contextVars.currentUser || {
  * @property {String} name
  **/
 function Comment(data) {
-    var self = this;
+        var self = this;
```

Comment:
Minor stylistic quibble, but this section can be outdented. (8 space indenting is a bit much for one level; our JS style guide recommends 4) 

http://cosdev.readthedocs.org/en/latest/style_guides/javascript.html